src/Compilers/CSharp/Portable/Binder/Binder_Operators.cs

```diff
@@ -2550,6 +2550,20 @@ private bool IsOperandErrors(CSharpSyntaxNode node, ref BoundExpression operand,
                     }
 
                     return true;
+
+                default:
+                    if ((object)operand.Type == null && !operand.IsLiteralNull())
+                    {
+                        if (!operand.HasAnyErrors)
+                        {
+                            // Operator 'is' cannot be applied to operand of type '(int, <null>)'
+                            diagnostics.Add(ErrorCode.ERR_BadUnaryOp, node.Location, "is", operand.Display);
```

Comment:
I'll change it to `IsToken.Text`, just in case we decide to change it in the future.

---
In reply to: [78599437](https://github.com/dotnet/roslyn/pull/13747#discussion_r78599437) [](ancestors = 78599437)